raiden/network/proxies/token_network.py

```diff
@@ -1134,7 +1134,7 @@ def _check_channel_state_for_withdraw(
     def _check_channel_state_for_settle(self, participant1, participant2, channel_identifier):
         channel_data = self.detail_channel(participant1, participant2, channel_identifier)
         if channel_data.state == ChannelState.SETTLED:
-            raise RaidenUnrecoverableError(
+            raise RaidenRecoverableError(
```

Comment:
Can you also change the message from:
```
  'Channel is not in a closed state. It cannot be settled',
```
to 
```
  'Channel is already settled.',
```

Reply:
sure. But won't run the tests again. So noting down that all integration tests passed before amending my commit.